x-pack/plugins/observability_solution/inventory/public/components/search_bar/entity_types_controls.tsx

```diff
@@ -8,19 +8,17 @@ import { EuiComboBox, EuiComboBoxOptionOption } from '@elastic/eui';
 import { css } from '@emotion/react';
 import { i18n } from '@kbn/i18n';
 import React from 'react';
-import { EntityType } from '../../../common/entities';
 import { useInventoryAbortableAsync } from '../../hooks/use_inventory_abortable_async';
 import { useInventoryParams } from '../../hooks/use_inventory_params';
 import { useKibana } from '../../hooks/use_kibana';
-import { getEntityTypeLabel } from '../../utils/get_entity_type_label';
 
 interface Props {
-  onChange: (entityTypes: EntityType[]) => void;
+  onChange: (entityTypes: string[]) => void;
 }
 
-const toComboBoxOption = (entityType: EntityType): EuiComboBoxOptionOption<EntityType> => ({
+const toComboBoxOption = (entityType: string): EuiComboBoxOptionOption<string> => ({
   key: entityType,
-  label: getEntityTypeLabel(entityType),
+  label: entityType,
```

Comment:
What's the plan here? is EEM going to send the label?

Reply:
That's the plan for the future.